docs/BasicCommunication/GetAppProperties/index.md

```diff
@@ -0,0 +1,88 @@
+## GetAppProperties
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Get the current properties of an application
+
+
+### Request
+
+#### Parameters
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|policyAppID|String|false|maxlength: "100"|
+
+
+### Response
+In case `policyAppID` is ommited in the request, SDL provides all app properties in the response.
+
+#### Parameters
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|properties|[Common.AppProperties](../../common/structs/#appproperties)|false|array: true<br>minsize: 1|
+
+
+### Example Request
+```json
+{
+  "id" : 47,
+  "jsonrpc" : "2.0",
+  "method" : "BasicCommunication.GetAppProperties",
+  "params" :
+  {
+    "policyAppID" : 123456
+  }
+}
+```
+
+### Example Response
+
+```json
+{
+  "id" : 47,
+  "jsonrpc" : "2.0",
+  "result" : {
+      "properties": {
```

Comment:
properties should be an array of objects

Reply:
please, see updates in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/187/commits/4ebba070bd78b95c3a00721cc2bcb639a6dfb422